pypeit/par/pypeitpar.py

```diff
@@ -834,15 +834,17 @@ def __init__(self, spec_method=None, spec_maxshift=None, spectrum=None,
         dtypes['multi_min_SN'] = [int, float]
         descr['multi_min_SN'] = 'Minimum S/N for analyzing sky spectrum for flexure'
 
-        defaults['excessive_shift'] = 'crash'
+        defaults['excessive_shift'] = 'use_another_value'
```

Comment:
does `use_another_value` capture what the code is doing? From reading the code, it seems like the median (or, rather, the 50th percentile) was being calculated. I wonder if something more descriptive could be used? Something like "median" or "use_median"?

Reply:
Also, should this be the default value? In other words, should we maybe crash, and suggest to the user what their options are if they want to re-run the reduction?

Reply:
Thanks! I changed it to `use_median` and explain it better in the description.

I prefer not to have `crash` as default because of the problem of different values in different machines. I think this option should make the shift calculation more stable, or at least not giving completely wrong values.